src/z3c/insist/enforce.py

```diff
@@ -49,7 +49,7 @@ def createStore(self, factory, path):
         return factory.fromRootAndFilename(self.root, path)
 
     def getStoreFromEvent(self, event):
-        path = watchdog.utils.unicode_paths.decode(event.src_path)
+        path = os.fsdecode(event.src_path)
```

Comment:
As in https://github.com/gorakhargosh/watchdog/blob/master/src/watchdog/events.py#L376